partner_risk_insurance/wizard/invoice_risk_insurance_wizard.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class InvoiceRiskInsuranceWizard(models.TransientModel):
+    _name = "partner.risk.insurance.wizard"
+    _description = "Wizard to see invoices sorted by risk"
+
+    date_since = fields.Date(
```

Comment:
Although the term used is correct, we usually use date_from and date_to to name these fields.
They also match better with the strings used.